lua/telescope/mappings.lua

```diff
@@ -111,51 +111,58 @@ local telescope_map = function(prompt_bufnr, mode, key_bind, key_func, opts)
   if opts.silent == nil then opts.silent = true end
 
   if type(key_func) == "string" then
-    a.nvim_buf_set_keymap(
+    key_func = actions[key_func]
+  elseif type(key_func) == "table" then
+    if key_func.type == "command" then
+      a.nvim_buf_set_keymap(
+        prompt_bufnr,
+        mode,
+        key_bind,
+        key_func[1],
+        opts or {
+          silent = true
+        }
+      )
+      return
+    elseif key_func.type == "action_key" then
+      key_func = actions[key_func[1]]
+    end
+  end
+
```

Comment:
Post here:

I only changed the indentation. From if-else to, if end and the `keymap` branch returns